src/resizing.jl

```diff
@@ -168,8 +168,16 @@ function restrict_indices(r::UnitRange)
 end
 
 # imresize
-imresize(original::AbstractArray, dim1, dimN...) = imresize(original, (dim1,dimN...))
+imresize(original::AbstractArray, dim1::T, dimN::T...) where T<:Union{Integer,AbstractUnitRange} = imresize(original, (dim1,dimN...))
+imresize(original::AbstractArray, short_size::AbstractArray{T,1}) where T <:Union{Integer,AbstractUnitRange} = imresize(original, (short_size...))
 
+function imresize(original::AbstractArray, ratio::Real)
```

Comment:
Needs to be a keyword argument, see https://github.com/JuliaImages/ImageTransformations.jl/issues/57#issuecomment-417214869.

Specifically, what if `ratio = 2`? That's not an `AbstractFloat` (so your `throw(MethodError(...))` below won't catch it) and yet my intention was to get a 2x change in size.

Reply:
This is why I put some note on the function document. But yes this would implicitly introduce the possibility of a new bug. (although not so hard to find)
I'll change this.